config/config.go

```diff
@@ -68,6 +68,7 @@ type ResourceType struct {
 
 type FilterRule struct {
 	NamesRegExp []Expression `yaml:"names_regex"`
+	// TODO(james): consider adding time filter rule here instead of passing in as function argument in other place.
```

Comment:
I like this todo idea, it always seemed weird how that got tossed into ever function call